environment_raven.yml

```diff
@@ -0,0 +1,29 @@
+name: synth_seg_py39
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9.7
+  - cudatoolkit=11.2.2
+  - cudnn=8.1.0
+  - pip:
+      - tensorflow-gpu==2.11.0
```

Comment:
This threw me off a bit. Why should we install tensorflow and the other packages through `pip` which has a much weaker resolve mechanism for identifying package dependency errors?

My usual approach is to install as many packages as possible through conda and only use pip as a last resort.

Reply:
Maybe a bad habit, but I tend to use pip as the package and conda as the "environment manager", simply because not all packages are available or up to date on conda. And I try to avoid a mixture, but there may be no good reason.
So let's go for the conda first approach, absolutely fine with me! :+1: